apps/AEPSampleApp/package.json

```diff
@@ -11,13 +11,13 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "@adobe/react-native-aepcore": "^1.0.0-alpha.2",
-    "@adobe/react-native-aepuserprofile": "^1.0.0-alpha.3",
-    "@adobe/react-native-aepedge": "^1.0.0-alpha.2",
-    "@adobe/react-native-aepedgeidentity": "^1.0.0-alpha.2",
-    "@adobe/react-native-aepedgeconsent": "^1.0.0-alpha.1",
-    "@adobe/react-native-aepmessaging": "^1.0.0-alpha.2",
-    "@adobe/react-native-aepassurance": "^3.0.0-alpha.0",
+    "@adobe/react-native-aepassurance": "^3.0.0-beta.1",
```

Comment:
Should we start with x-beta.0?

Reply:
For Assurance we also need to update the Readme: https://github.com/adobe/aepsdk-react-native/tree/staging/packages/assurance#install-npm-package

Reply:
https://www.npmjs.com/package/@adobe/react-native-acpcore
https://www.npmjs.com/package/@adobe/react-native-aepcore
We used to release the alpha/beta version start from `.1`. Let me know if there is any reason we need to release a `beta.0` instead of a `beta.1`.

Reply:
For Assurance, it's still a private package (not being released with other extensions). We can do more cleanup for Assurance in a separate PR after @PravinPK has updated the Gitbook.

Reply:
👍  
saw some occurrences with alpha.0 so I thought we can follow the same, but beta.1 sounds good then